code/game/objects/items/weapons/tools.dm

```diff
@@ -34,6 +34,13 @@
 	playsound(loc, 'sound/weapons/genhit.ogg', 50, 1, -1)
 	return (BRUTELOSS)
 
+/obj/item/weapon/wrench/golden
+	name = "golden wrench"
+	desc = "Destroying one of these makes fat neckbeards around the galaxy cry."

```

Comment:
No I'm just implying that it falls in that weird little crevasse of a game's existence where a game is fairly old but not yet "nostalgic".